indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -425,4 +440,108 @@ public static Path prependFSIfNullScheme(FileSystem fs, Path path)
     }
     return path;
   }
+
+  // TODO: Replace this whenever hadoop gets their act together and stops breaking with more recent versions of Guava
+  public static long unzipNoGuava(
```

Comment:
Zero, it is actually because `com.metamx.common.CompressionUtils` uses byte source and byte sink in the "useful" methods. Also `com.google.common.io.ByteStreams#copy(java.io.InputStream, java.io.OutputStream)` is used quite a bit in there.